spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/types/ColumnType.scala

```diff
@@ -157,7 +153,13 @@ object ColumnType {
   }
 
   /** Returns natural Cassandra type for representing data of the given Scala type */
-  def fromScalaType(dataType: Type): ColumnType[_] = {
+  def fromScalaType(
+    dataType: Type,
+    protocolVersion: ProtocolVersion = ProtocolVersion.NEWEST_SUPPORTED): ColumnType[_] = {
+
+    import ProtocolVersion._
+    def versionGTE( targetProtocolVersion: ProtocolVersion) : Boolean =
```

Comment:
not very scala-ish... :)
maybe a value class with > and  < methods would be better?